test/e2e/common/setup-kubestellar-ocp.sh

```diff
@@ -51,27 +51,32 @@ kflex create its1 --type vcluster -p ocm $disable_chatty_status
 
 :
 : -------------------------------------------------------------------------
-: Install singleton status return addon in IMBS1
+: Install singleton status return addon in ITS1
 :
 wait-for-cmd kubectl --context its1 api-resources "|" grep managedclusteraddons
 helm --kube-context its1 upgrade --install status-addon -n open-cluster-management oci://ghcr.io/kubestellar/ocm-status-addon-chart --version v${OCM_STATUS_ADDON_VERSION}
 :
 : -------------------------------------------------------------------------
 : Create a Workload Description Space wds1 directly in KubeFlex.
 :
-kflex create wds1 -p kubestellar
+kflex create wds1 -p kubestellar $disable_chatty_status
+kubectl --context kscore label cp wds1 kflex.kubestellar.io/cptype=wds
+
 echo "wds1 created."
 
 :
 : -------------------------------------------------------------------------
 : Run OCM transport controller in a pod
 :
-kflex ctx
-bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/deploy-transport-controller.sh) wds1 its1
-wait-for-cmd '(kubectl -n wds1-system wait --for=condition=Ready pod/$(kubectl -n wds1-system get pods -l name=transport-controller -o jsonpath='{.items[0].metadata.name}'))'
+helm --kube-context kscore upgrade --install ocm-transport-plugin oci://ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin --version ${OCM_TRANSPORT_PLUGIN} \
+ --set transport_cp_name=its1 \
+ --set wds_cp_name=wds1 \
+ -n wds1-system
```

Comment:
This is a step in the wrong direction, getting even further away from testing the local copy of the repo. Not a blocker for this PR, #2071 predates this PR.

Reply:
@ezrasilvera addressed this comment in this #2071

Reply:
And I responded to his remark.